### PR TITLE
Updates log categorization docs

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR updates the screenshots and the text of the `Categorize log entries` page.

### CODE REVIEW

File: docs/en/observability/categorize-logs.asciidoc [hunk 1]
```diff
@@ -6,53 +6,55 @@ log messages are the same or very similar, so classifying them can reduce
...
 [[create-log-categories]]
 == Create log categories
 
-Create a machine learning job to categorize log messages automatically. Machine learning will observe
-the static parts of the message, cluster similar messages, and classify them into message categories.
+Create a 
+{ml-docs}/ml-configuring-categories.html[{ml} job to categorize log messages] 
+automatically. {ml-cap} observes the static parts of the message, cluster 
+similar messages, classify them into message categories, and detect unusually 
+high message counts in the categories.
```
Comment (EamonnTP) [inline docs/en/observability/categorize-logs.asciidoc:21]:
```suggestion
Create a 
{ml-docs}/ml-configuring-categories.html[{ml} job to categorize log messages] 
automatically. {ml-cap} observes the static parts of the message, clusters 
similar messages, classifies them into message categories, and detects unusually 
high message counts in the categories. 
```

File: docs/en/observability/categorize-logs.asciidoc [hunk 2]
```diff
@@ -6,53 +6,55 @@ log messages are the same or very similar, so classifying them can reduce
...
+{ml-docs}/ml-configuring-categories.html[{ml} job to categorize log messages] 
+automatically. {ml-cap} observes the static parts of the message, cluster 
+similar messages, classify them into message categories, and detect unusually 
+high message counts in the categories. 
+
+[role="screenshot"]
+image::images/log-create-categorization-job.jpg[Configure log categorization job]
+
+1. Select *Categories*, and you are prompted to use {ml}, which carries out the 
+   log rate categorizations.
```
Comment (EamonnTP) [inline docs/en/observability/categorize-logs.asciidoc:27]:
```suggestion
1. Select *Categories*, and you are prompted to use {ml} to create 
   log rate categorizations.
```

File: docs/en/observability/categorize-logs.asciidoc [hunk 3]
```diff
@@ -6,53 +6,55 @@ log messages are the same or very similar, so classifying them can reduce
...
+2. Choose a time range for the {ml} analysis. By default, the {ml} job analizes 
+   log messages no lder than four weeks and continues indefinitely.
```
Comment (EamonnTP) [inline docs/en/observability/categorize-logs.asciidoc:29]:
```suggestion
2. Choose a time range for the {ml} analysis. By default, the {ml} job analyzes 
   log messages no older than four weeks and continues indefinitely.
```

File: docs/en/observability/categorize-logs.asciidoc [hunk 4]
```diff
@@ -6,53 +6,55 @@ log messages are the same or very similar, so classifying them can reduce
...
+3. Add the indices that contain the logs you want to analyze.
```
Comment (EamonnTP) [inline docs/en/observability/categorize-logs.asciidoc:30]:
```suggestion
3. Add the indices that contain the logs you want to examine.
```

File: docs/en/observability/categorize-logs.asciidoc [hunk 5]
```diff
@@ -6,53 +6,55 @@ log messages are the same or very similar, so classifying them can reduce
...
+4. Click *Create ML job*. The job is created and it starts run. It takes a few
```
Comment (EamonnTP) [inline docs/en/observability/categorize-logs.asciidoc:31]:
```suggestion
4. Click *Create ML job*. The job is created, and it starts to run. It takes a few 
```

File: docs/en/observability/categorize-logs.asciidoc [hunk 6]
```diff
@@ -6,53 +6,55 @@ log messages are the same or very similar, so classifying them can reduce
...
 
-Here we see the *Categories* page, which lists various log categories from
-`elasticsearch.server` logs, along with the maximum anomaly score on the right.
+The *Categories* page lists all the log categories from the selected indices. 
+You can filter the categories by indices. The screenshot below shows the 
+categories from the `elastic.agent` log.
 
 [role="screenshot"]
-image::images/log-categories.png[Log categories]
+image::images/log-categories.jpg[Log categories]
```
Comment (EamonnTP) [inline docs/en/observability/categorize-logs.asciidoc:44]: Should this not be `.png`?

File: docs/en/observability/categorize-logs.asciidoc [hunk 7]
```diff
@@ -6,53 +6,55 @@ log messages are the same or very similar, so classifying them can reduce
...
 
-The highlighted log category has a very high anomaly score, and looks like this:
+The category row contains the following information:
```
Comment (EamonnTP) [inline docs/en/observability/categorize-logs.asciidoc:46]: You need an extra line below for the bullet points to render properly.

File: docs/en/observability/categorize-logs.asciidoc [hunk 8]
```diff
@@ -6,53 +6,55 @@ log messages are the same or very similar, so classifying them can reduce
...
+categories from the `elastic.agent` log.
 
 [role="screenshot"]
-image::images/log-categories.png[Log categories]
+image::images/log-categories.jpg[Log categories]
 
-The highlighted log category has a very high anomaly score, and looks like this:
+The category row contains the following information: 
+* message count: shows how many messages belong to the given category.
+* trend: indicates how the occurrance of the messages changes in time.
```
Comment (EamonnTP) [inline docs/en/observability/categorize-logs.asciidoc:48]:
```suggestion
* trend: indicates how the occurrence of the messages changes in time.
```

File: docs/en/observability/categorize-logs.asciidoc [hunk 9]
```diff
@@ -6,53 +6,55 @@ log messages are the same or very similar, so classifying them can reduce
...
-[source,console-result]
-----------------------------------
-⁕type⁕server⁕timestamp⁕level⁕component⁕o.e.m.j.JvmGcMonitorService⁕cluster.name
-⁕elasticsearch⁕node.name⁕message⁕gc⁕overhead⁕spent⁕collecting⁕in⁕the⁕last⁕cluster.uuid⁕
-jx_Bw17eT_OQshT2IrDuzg⁕node.id⁕
-----------------------------------
+You can view the individual log messages under a particular category by clicking 
+the arrow at the end of the row. To further examine an individual message, it 
+can be viewed in the corresponding log event on the *Stream* page or can be 
+displayed in its context.
```
Comment (EamonnTP) [inline docs/en/observability/categorize-logs.asciidoc:57]:
```suggestion
To view al log message under a particular category, click  
the arrow at the end of the row. To further examine a message, it 
can be viewed in the corresponding log event on the *Stream* page or displayed in its context.
```

**Review by EamonnTP — CHANGES_REQUESTED** [review]
Thanks @szabosteve looks great! Just a few minor suggestions from me.

File: docs/en/observability/categorize-logs.asciidoc (hunk 7 above)
Comment (szabosteve) [reply docs/en/observability/categorize-logs.asciidoc:46]:
```suggestion
The category row contains the following information: 

```

File: docs/en/observability/categorize-logs.asciidoc (hunk 6 above)
Comment (szabosteve) [reply docs/en/observability/categorize-logs.asciidoc:44]: I pushed a log-categories.jpg in this PR along with the textual changes, so the extension of this particular image is correct here.

**Review by EamonnTP — APPROVED** [review]

**Pull request merged**